File: src/com/esielkar/builder/Product.java
package com.esielkar.builder;

public class Product {
    private boolean a;
    private boolean b;
    private boolean c;

    public void setA(boolean a) {
        this.a = a;
    }

    public void setB(boolean b) {
        this.b = b;
    }

    public void setC(boolean c) {
        this.c = c;
    }

    @Override
    public String toString() {
        return "A: " + a + " B: " + b + " C: " + c;
    }

}

File: dart/design_patterns/bin/builder/product.dart
class Product {
  bool _a;
  bool _b;
  bool _c;

  Product()
      : _a = false,
        _b = false,
        _c = false;

  set a(bool a) {
    _a = a;
  }

  set b(bool b) {
    _b = b;
  }

  set c(bool c) {
    _c = c;
  }

  @override
  String toString() => "A: $_a  B: $_b  C: $_c";
}
